catalyst-generator/src/main/java/com/intellecteu/catalyst/generator/ProjectGenerator.java

```diff
@@ -342,19 +346,25 @@ private void appendRouter(String usecaseName, File srcDir, Map<String, Object> m
     }
   }
 
+  private void appendDefaultProperties(StringBuilder appProperties) throws IOException {
+    try (InputStream defaultPropsFile = ResourceUtils
+        .getURL("classpath:templates/camel/properties/default.yml").openStream();) {
+      String defaultProperties = CharStreams.toString(
+          new InputStreamReader(defaultPropsFile, Charsets.UTF_8));
+      appProperties.append(defaultProperties).append(System.lineSeparator());
+    } catch (FileNotFoundException ex) {
+      log.warn("Default property file not found: {}", ex.getMessage());
```

Comment:
In such case better to let the app crash. Default properties should not be missing

Reply:
Fixed